salt/states/virt.py

```diff
@@ -52,20 +53,53 @@ def keys(name, basepath='/etc/pki'):
     basepath
         Defaults to ``/etc/pki``, this is the root location used for libvirt
         keys on the hypervisor
+
+    The following parameters are optional:
+
+        country
+            The country that the certificate should use.  Defaults to US.
+
+        state
+            The state that the certificate should use.  Defaults to Utah.
+
+        locality
+            The locality that the certificate should use.
+            Defaults to Salt Lake City.
+
+        organization
+            The organization that the certificate should use.
+            Defaults to Salted.
+
+        expiration_days
+            The number of days that the certificate should be valid for.
+            Defaults to 365 days (1 year)
+
     '''
-    #libvirt.serverkey.pem
-    #libvirt.servercert.pem
-    #libvirt.clientkey.pem
-    #libvirt.clientcert.pem
-    #libvirt.cacert.pem
+    # libvirt.serverkey.pem
+    # libvirt.servercert.pem
+    # libvirt.clientkey.pem
+    # libvirt.clientcert.pem
+    # libvirt.cacert.pem
```

Comment:
Not needed, but much like the line too long warnings are caught via the lint plugin I'm using so they show up as errors/warnings.  We're not failing on them but it makes for more readable code so I tend to fix them when I find them.